src/main/java/base/MainProgram.java

```diff
@@ -26,8 +26,15 @@ private MainProgram() {
      * @param args commandline arguments.
      */
     public static void main(String[] args) {
-        //Just use this project for now (src), will have to change
-        //to the target project with the gradle stuff
+        // Just use this project for now (src), will have to change
+        // to the target project with the gradle stuff
+        // List<CompilationUnit> compUnits = ProjectParser.projectToAst("src/test");
+        // System.out.println(VariableReader.readVariables(compUnits.get(1)));
+        // SingletonVerifier verifier = new SingletonVerifier();
+        // System.out.println("Passing Mock: " + verifier.verify(compUnits.get(2)));
+        // System.out.println(
+        //     "Passing Mock with multiple constructors: " + verifier.verify(compUnits.get(0)));
+        // System.out.println("Failing Mock: " + verifier.verify(compUnits.get(1)));
```

Comment:
Vi hittade mock grejer här i Main som vi kommenterade ut tills vidare

Reply:
Removed comments aswell, thanks for pointing this out!